packages/gradle/src/utils/get-gradle-report.ts

```diff
@@ -0,0 +1,169 @@
+import { readFileSync } from 'node:fs';
+import { join, relative } from 'node:path';
+
+import { workspaceRoot } from '@nx/devkit';
+
+import { execGradle } from './exec-gradle';
+
+let gradleReport: ReturnType<typeof processProjectReports>;
+export function getGradleReport() {
+  if (gradleReport) {
+    return gradleReport;
+  }
```

Comment:
This cache may live too long when daemon is active, can you validate that it reflects appropriate changes when a file is updated?

Reply:
add a function to delete cache